### PR TITLE
added invalid credentials

### PR DESCRIPTION
**Pull request by cmgonzales**
### What was the feature/problem?

Failed login could use a friendlier error message #72

### How does this PR address the above feature/problem?

Changing the state of a wrong credentials to a friendlier credentials

### Check lists (check `x` in `[ ]` of list items)

- [ x] Test passed
- [ ] Coding style (indentation, etc)

### Additional Comments (if any)

{Please write here}

### CODE REVIEW

File: ui/src/login/ValidatedSignInForm.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ export default class ValidatedSignInForm extends Component {
       if (response.status === 200) this.props.history.push('/landing')
       else {
         this.setState({
-          invalidFormErrorMsg: response.message,
+          invalidFormErrorMsg: "I'm sorry your credentials is not valid. Please try again",
```
Comment (crstamps2) [inline ui/src/login/ValidatedSignInForm.js]: Should be "are not valid", not "is not valid".  A designer may have an opinion on what this messaging should be. We have to keep in mind that majority of this experience is done by a kid. 

Also, catch your fork up with master to skip the GAE deploy check.

Comment (cmgonzales) [reply ui/src/login/ValidatedSignInForm.js]: Got it. I will make that change. Just want to confirm that a kid/kids will be the end user of this product and they have to understand the invalidFormerrorMsg

Comment (crstamps2) [reply ui/src/login/ValidatedSignInForm.js]: @hellonoosh what do you want to do here?

**Comment by aclaire-thd** [comment]
Currently merged code has a friendlier error message :)
"Incorrect username/password. Please try again."
MakeAWish-InvalidLogin-FriendlyErrorMessage

**Comment by aclaire-thd** [comment]
Looks like is covered [in another PR](https://github.com/homedepot/infinite-wish-board/pull/90) that was merged, so I'm closing this one.

**Pull request closed without merging**